### DISCUSSION TITLE
Photos from Storage are not showing

### DISCUSSION THREAD

**Discussion by krystianwojda** [supabase/supabase#28161] [Questions]
Hello,
I have smaller problem but I'dont have any idea where is a bug. I have component detail where I fetch data form supabase. All data good fetch but I have problem witch storage, beacuse photo don't showing. I don't have any communicat in console. 

name my buckets: ,,uploads''

`<template>
  <div v-if="isLoading">
    <Spinner />
  </div>
  <div v-else>
    <div class="complaint-detail">
      <h1>Szczegóły reklamacji</h1>
      <p><strong>Imię:</strong> {{ complaint.name }}</p>
      <p><strong>Nazwisko:</strong> {{ complaint.surname }}</p>
      <p><strong>ID:</strong> {{ complaint.id }}</p>
      <p><strong>Email:</strong> {{ complaint.email }}</p>
      <p><strong>Data złożenia:</strong> {{ complaint.date }}</p>
      <p><strong>Status:</strong> {{ complaint.status }}</p>
      <div v-if="imageUrls.length">
        <h2>Zdjęcia</h2>
        <div class="complaint-images">
          <img v-for="(url, index) in imageUrls" :key="index" :src="url" alt="Zdjęcie reklamacji" />
        </div>
      </div>
    </div>
  </div>
</template>

<script setup>
import { ref, onMounted } from 'vue';
import { useRoute } from 'vue-router';
import { createClient } from '@supabase/supabase-js';

const route = useRoute();

// Konfiguracja Supabase
const supabaseUrl = 'xyz';
const supabaseKey = 'xyz';
const supabase = createClient(supabaseUrl, supabaseKey);

const complaint = ref(null);
const imageUrls = ref([]);
const isLoading = ref(true);

const fetchComplaint = async (id) => {
  const { data, error } = await supabase
      .from('reklamacje')
      .select('*')
      .eq('id', id)
      .single();

  if (error) {
    console.error('Błąd podczas pobierania szczegółów reklamacji:', error);
  } else {
    complaint.value = data;

    // Pobieranie URL-i obrazów z Supabase Storage
    if (data.imagePaths) {
      const { data: imagePaths, error: imageError } = await supabase
          .storage
          .from('uploads') 
          .list(data.imagePaths);

      if (imageError) {
        console.error('Błąd podczas pobierania obrazów:', imageError);
      } else {
        imageUrls.value = imagePaths.map(imagePath => {
          return supabase.storage.from('uploads').getPublicUrl(imagePath.path).publicURL;
        });
      }
    }
  }
  isLoading.value = false;
};

onMounted(() => {
  const id = route.params.id;
  fetchComplaint(id);
});
</script>`

**Comment by GaryAustin1** [comment]
Not really clear what you are doing, or your exact issue.
Normally if you are not seeing things in storage you expect, your RLS settings are incorrect.
